### PR TITLE
New series details view

### PR DESCRIPTION
**Pull request by cromefire**
Navigation is funky right now and I don't know how to fix it, but the basic layout is there

### CODE REVIEW

**Comment by thornbill** [comment]
CI is failing to build with the following errors:

```
  The first 3 errors (out of 31) were:
  /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/BaseDetailsFragment.kt:37: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
    rowSelector.addClassPresenter(DetailsOverviewRow::class.java, DetailsOverviewPresenter(context!!))
                                                                                           ~~~~~~~~~
  /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/EpisodeDetailsFragment.kt:30: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
     ResumeAction(context!!, item),
                  ~~~~~~~~~
  /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/EpisodeDetailsFragment.kt:31: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
     PlayFromBeginningAction(context!!, item),
                             ~~~~~~~~~
```

File: app/build.gradle.kts [hunk 1]
```diff
@@ -1,98 +1,105 @@
 plugins {
-	id("com.android.application")
-	id("kotlin-android")
-	id("kotlin-android-extensions")
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-android-extensions")
```
Comment (thornbill) [inline app/build.gradle.kts:4]: Why all the indentation changes here?

Comment (cromefire) [reply app/build.gradle.kts:4]: Android studio did that automatically because it seems that it wasn't formatted correctly

**Comment by cromefire** [comment]
> CI is failing to build with the following errors:
> 
> ```
>   The first 3 errors (out of 31) were:
>   /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/BaseDetailsFragment.kt:37: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
>     rowSelector.addClassPresenter(DetailsOverviewRow::class.java, DetailsOverviewPresenter(context!!))
>                                                                                            ~~~~~~~~~
>   /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/EpisodeDetailsFragment.kt:30: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
>      ResumeAction(context!!, item),
>                   ~~~~~~~~~
>   /home/vsts/work/1/s/app/src/main/java/org/jellyfin/androidtv/details/fragments/EpisodeDetailsFragment.kt:31: Error: Use requireContext() instead of context!! [UseRequireInsteadOfGet]
>      PlayFromBeginningAction(context!!, item),
>                              ~~~~~~~~~
> ```

Seems like somebody fixed the lint or something like that

**Comment by cromefire** [comment]
Build will probably on rerun because of the draft PR

**Comment by cromefire** [comment]
Just too much magic involved

**Pull request closed without merging**